src/System/God/GodDoodle.ZC

```diff
@@ -0,0 +1,192 @@
+#help_index "God;Graphics/Sprite;Sprites"
+
+U0 GodDoodleDraw(CTask *task,CDC *dc)
+{
+  GrBlot(dc,0,0,god.doodle_dc);
+  if (Blink) {
+    if (god.doodle_done) {
+      dc->color=RED;
+      GrPrint(dc,(task->pix_width-FONT_WIDTH*29)>>1,
+	    (task->pix_height-3*FONT_HEIGHT)>>1,
+	    "Press <ESC> to insert sprite.");
+      GrPrint(dc,(task->pix_width-FONT_WIDTH*39)>>1,
+	    (task->pix_height-3*FONT_HEIGHT)>>1+2*FONT_HEIGHT,
+	    "Press <SHIFT-ESC> to throw-away sprite.");
+    } else {
+      dc->color=GREEN;
+      GrPrint(dc,(task->pix_width-FONT_WIDTH*25)>>1,
+	    (task->pix_height-FONT_HEIGHT)>>1,
+	    "Press <SPACE> repeatedly.");
+    }
+  }
+}
+
+U0 GodDoodleSmooth(I64 num)
+{
+  CDC *dc=DCExt(god.doodle_dc,0,0,
+	god.doodle_dc->width-1,god.doodle_dc->height-1);
+  I64 i,x,y,x1,y1,c,histogram[16],best,best_cnt,c_old=god.doodle_dc->color;
+  for (y=0;y<god.doodle_dc->height;y++)
+    for (x=0;x<god.doodle_dc->width;x++) {
+      MemSet(histogram,0,sizeof(histogram));
+      for (y1=y-num;y1<=y+num;y1++)
+	for (x1=x-num;x1<=x+num;x1++) {
```

Comment:
the indent of this for loop is wrong

see:
https://github.com/cia-foundation/TempleOS/blob/c26482bb6ad3f80106d28504ec5db3c6a360732c/Adam/God/GodDoodle.HC#L32-L37